src/api/provisionFormOptionsData.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	provisionFormOptions: 
+	[
+		{
+			services: [
+				{
+					identifier: 'github'
+				}
+			],
+			automationSoftwares: [
+				'ansible'
+			],
+			serverProviders: [
+				'digitalOcean'
+			],
+			distributions: [
+				'debian'
+			],
+			applications: [
+				{
+					identifier: 'Ghost',
+					name: 'Ghost'
+				},
+				{
+					identifier: 'GitLab',
+					name: 'GitLab'
```

Comment:
AFAIK name is displayed and `identifier` is entirely for us, right @iLeonelPerea ? If this is so, name would be something that looks pretty and `identfier` would follow the snake_case convention...